test/docx/body-reader.tests.js

```diff
@@ -141,6 +141,18 @@ test("numbering properties are ignored if w:numId is missing", function() {
     assert.equal(numberingLevel, null);
 });
 
+test("stores instrText returns empty result", function() {
+    var instrTextXml = new XmlElement("w:instrText", {}, [xml.text(' HYPERLINK "http://example.com"')]);
+    var instrText = readXmlElementValue(instrTextXml);
+    assert.deepEqual(instrText, []);
+});
+
+test("run is wrapped in a hyperlink if the runStyle is Hyperlink", function() {
+    var hyperlink = createHyperlink("http://example.com");
+    assert.equal(hyperlink.type, 'hyperlink');
+    assert.equal(hyperlink.href, 'http://example.com');
+});
+
```

Comment:
I think there's a test case missing for ensuring that elements after the end don't get turned into hyperlinks.